m4_sessions/1901-inning/transforming-narrative.md

```diff
@@ -37,6 +37,30 @@ ALL 5 Minutes - We will open up the conversation so that students can share thei
 
 Know that this experience has prepared you for failure. Being mindful of this failure and accepting of the fact that we can grow from our failures will allow us to perservere and succeed. 
 
+## New Ideas
+
+Introduce fixed and growth mindsets and then give students scenarios and ask them to decide if each is a fixed mindset or a growth mindset.
+
+Assign groups based on mixing together introverts and extroverts. Use cards that are color coded by how introverted or extroverted the students are then sort them into piles for each of us to run as small groups with a mixture of each color.
```

Comment:
This would be so great!  I think often times working style becomes a fixed mindset.